### DISCUSSION TITLE
Change Filament in new Version

### DISCUSSION THREAD

**Discussion by vaderag** [Jyers/Marlin#349] [Q&A] [answered]
I just updated to the latest version (standard, no leveling) and noticed that the Change Filament option has appeared in the menu
I was of the understanding that this needed manual compilation to get added due to some specific settings - has that changed?

Does this mean I can now use M600?

**Comment by Jyers** [comment, marked as answer]
It _did_ need configuration, but I was able to now move those setting to the on screen menu! In Control>Advanced there are two settings for the load and unload distances. Once those are set you will be able to use M600 M701 and M702!
